.circleci/config.yml

```diff
@@ -85,6 +85,7 @@ workflows:
     jobs:
       - run-specs:
           name: 'run-specs-ruby-<<matrix.ruby>>-db-<<matrix.database>>'
+          context: stripe-test-credentials
```

Comment:
We'll probably have unauthorized builds for contributions, as we experienced when configuring Slack alerts 😢 Not for now, but maybe we can use vcr in the future?